test/integration/targets/eos_lag_interfaces/tests/cli/deleted.yaml

```diff
@@ -1,9 +1,10 @@
 ---
-- include_tasks: reset_config.yml
+- include_tasks: _reset_config.yml
```

Comment:
I'm confused by what's going on here... you've added a file named `reset_config.yaml`, but including `_reset_config.yml`?